dgb: Simnet harness fails livetest.

I tested [the new digibyte harness code](https://github.com/decred/dcrdex/pull/3391) with `client/asset/dgb/regnet.go` which uses 2 wallets (alpha & gamma)

```sh
$ go test -v -count=1 -tags=harness -run=TestWallet

=== RUN   TestWallet
    livetest.go:223: Setting up alpha/beta/gamma wallet backends...
2025-10-23 16:25:50.645 [DBG] TEST[alpha.][RPC]: Node at 127.0.0.1:20766 reports subversion "/DigiByte:8.26.0/"
2025-10-23 16:25:50.652 [INF] TEST[alpha.]: Connected wallet with current best block 56e20746016f681261c03195ad9b8ab676f78753b03036fbe99f16af58ba0dba (131)
2025-10-23 16:25:50.652 [DBG] TEST[alpha.]: Using tx history db at /tmp/3325947744/first/regtest/txhistorydb-44eec75aeec4015f3842fe35e453dc3bc2aaf7c2
2025-10-23 16:25:50.680 [DBG] TEST[alpha.gamma][RPC]: Node at 127.0.0.1:20766 reports subversion "/DigiByte:8.26.0/"
peer count = 1, err = <nil>2025-10-23 16:25:50.693 [INF] TEST[alpha.gamma]: Connected wallet with current best block 56e20746016f681261c03195ad9b8ab676f78753b03036fbe99f16af58ba0dba (131)
2025-10-23 16:25:50.694 [DBG] TEST[alpha.gamma]: Using tx history db at /tmp/3325947744/second/regtest/txhistorydb-9eae1dc822001f920450892bdf57d76b983dfb23
peer count = 1, err = <nil>2025-10-23 16:25:50.819 [INF] TEST: Wallets configured
2025-10-23 16:25:50.820 [INF] TEST: first 791957.873100 available, 7200000.000000 immature, 0.000000 locked
2025-10-23 16:25:50.822 [INF] TEST: second 216.000000 available, 0.000000 immature, 0.000000 locked
2025-10-23 16:25:50.822 [INF] TEST: Testing FundOrder
2025-10-23 16:25:50.822 [DBG] TEST[alpha.gamma]: Attempting to fund order for 0.06 dgb, maxFeeRate = 100, max swaps = 6

    livetest.go:286: Funding error: error funding swap value of 0.06: not enough to cover requested funds. 0 available in 0 UTXOs

2025-10-23 16:25:50.825 [ERR] TEST[alpha.]: error identifying transaction: rawrequest (gettransaction) error: request was canceled by the caller
2025-10-23 16:25:50.877 [ERR] TEST[alpha.gamma]: error identifying transaction: rawrequest (gettransaction) error: request was canceled by the caller
--- FAIL: TestWallet (0.27s)
FAIL
```
The bash harness makes 4 wallets alpha, beta, delta, gamma and then alpha funds them but what I see is 

```sh
dev@dev-Thin-15-B12UCX ~/dextest/dgb/harness-ctl $ ./alpha getbalance
791957.87310000
dev@dev-Thin-15-B12UCX ~/dextest/dgb/harness-ctl $ ./beta getbalance
0.00000000
dev@dev-Thin-15-B12UCX ~/dextest/dgb/harness-ctl $ ./delta getbalance
0.00000000
dev@dev-Thin-15-B12UCX ~/dextest/dgb/harness-ctl $ ./gamma getbalance
0.00000000
```

--- a/client/asset/bch/regnet_test.go
+++ b/client/asset/bch/regnet_test.go
@@ -2,7 +2,8 @@
 
 package bch
 
-// Regnet tests expect the BCH test harness to be running.
+// Regnet tests expect the BCH test harness to be running. The harness miner
+// must be OFF.
 //
 // Sim harness info:
 // The harness has three wallets, alpha, beta, and gamma.

--- a/client/asset/btc/livetest/livetest.go
+++ b/client/asset/btc/livetest/livetest.go
@@ -1,6 +1,7 @@
 package livetest
 
-// Regnet tests expect the BTC test harness to be running.
+// Regnet tests expect the BTC test harness to be running. The harness miner
+// must be OFF.
 //
 // Sim harness info:
 // The harness has three wallets, alpha, beta, and gamma.

--- a/client/asset/dgb/regnet_test.go
+++ b/client/asset/dgb/regnet_test.go
@@ -2,7 +2,13 @@
 
 package dgb
 
-// Regnet tests expect the DGB test harness to be running.
+// Regnet tests expect the DGB test harness to be running. The harness miner
+// must be OFF.
+//
+// NOTE: After mining blocks, DigiByte node must do 2 things so tests will not
+// pass until the harness is done with:
+//  1. Update Wallet DB (fast): Write transaction to wallet database
+//  2. Update UTXO Index (slow): Rebuild index of all unspent outputs
 //
 // Sim harness info:
 // The harness has three wallets, alpha, beta, and gamma.

--- a/client/asset/doge/regnet_test.go
+++ b/client/asset/doge/regnet_test.go
@@ -2,7 +2,8 @@
 
 package doge
 
-// Regnet tests expect the DOGE test harness to be running.
+// Regnet tests expect the DOGE test harness to be running. The harness miner
+// must be OFF.
 
 import (
 	"context"

--- a/client/asset/firo/regnet_test.go
+++ b/client/asset/firo/regnet_test.go
@@ -2,7 +2,8 @@
 
 package firo
 
-// Regnet tests expect the Firo test harness to be running.
+// Regnet tests expect the Firo test harness to be running. The harness miner
+// must be OFF.
 //
 // Sim harness info:
 // The harness has four nodes: alpha, beta, gamma and delta with one wallet

--- a/client/asset/ltc/regnet_test.go
+++ b/client/asset/ltc/regnet_test.go
@@ -2,7 +2,8 @@
 
 package ltc
 
-// Regnet tests expect the LTC test harness to be running.
+// Regnet tests expect the LTC test harness to be running. The harness miner
+// must be OFF.
 //
 // Sim harness info:
 // The harness has three wallets, alpha, beta, and gamma.

--- a/client/asset/zcl/regnet_test.go
+++ b/client/asset/zcl/regnet_test.go
@@ -2,7 +2,8 @@
 
 package zcl
 
-// Regnet tests expect the ZEC test harness to be running.
+// Regnet tests expect the ZEC test harness to be running. The harness miner
+// must be OFF.
 
 import (
 	"testing"

--- a/client/asset/zec/regnet_test.go
+++ b/client/asset/zec/regnet_test.go
@@ -2,7 +2,8 @@
 
 package zec
 
-// Regnet tests expect the ZEC test harness to be running.
+// Regnet tests expect the ZEC test harness to be running. The harness miner
+// must be OFF.
 
 import (
 	"context"

--- a/dex/testing/dgb/README.md
+++ b/dex/testing/dgb/README.md
@@ -1,0 +1,7 @@
+## Digibyte harness
+
+Important: After mining blocks, DigiByte node must do 2 things:
+- Update Wallet DB (fast): Write transaction to wallet database
+- Update UTXO Index (slow): Rebuild index of all unspent outputs
+
+Do not use until it is ready.

